Dockerfile

```diff
@@ -36,18 +44,15 @@ USER composer
 # Run Composer Install
 RUN /usr/local/bin/composer install
 
-# Generate App Key
-RUN php artisan key:generate
```

Comment:
After creating a `.env` file copied from the `.env.example` file, you still need to set up an App key. Maybe if you bring the `cp` command to this point, before generating the key, it will fix it?

Reply:
See my response to your other comment: https://github.com/erikaheidi/hacktober-board/pull/32#discussion_r339569070

I will try this for now and if it works I'll update the branch and we can decide whether to leave it as-is for now or if we would rather pull the key generation into an entrypoint script.

Reply:
I just realized I'm already running the key generation after the `cp` command. See here:
https://github.com/erikaheidi/hacktober-board/pull/32/files#diff-3254677a7917c6c01f55212f86c57fbfR54

Is that what you meant? Or am I misunderstanding something?

If this is correct, then I've tested it this way and everything seems to work fine.

Reply:
yes that's exactly it, sorry I didn't notice it was there. thank you, I'll merge it now!